docs/collection.md

```diff
@@ -46,16 +47,17 @@ $collection->someMethod('a value', function(ElementType $element, $query) {
     return $element->getSomething() === $query;
 });
 ```
+
 !!! note
-    Be careful: the parameters order is mandatory.
-    
+Be careful: the parameters order is mandatory.
```

Comment:
An admonition message must be indented with 4 spaces (or 1 tab) otherwise it's not included into the admonition but it's written on the next  line.